apps/src/dance/lottie/LottieDancerUtils.ts

```diff
@@ -796,11 +797,66 @@ export async function mirrorPngDataUrl(dataUrl: string): Promise<string> {
   return canvas.toDataURL('image/png');
 }
 
+// Head crop can be overridden via DCDO or URL param.
+function getHeadCrop(): number {
+  const dcdoHeadCrop = DCDO.get('ai-dancer-head-crop');
+
+  if (dcdoHeadCrop === false) {
+    return Number(getConfigValue('headCrop')) || 10;
```

Comment:
nit: I think this might prevent a weird edge case where 0 || 10 would give you 10? Probably doesn't matter but maybe an improvement?

> if (dcdoHeadCrop === false) {
>     const fromParam = Number(getConfigValue('headCrop'));
>     return Number.isFinite(fromParam) ? fromParam : 10;
>   }
> 
>   const n = Number(dcdoHeadCrop);
>   return Number.isFinite(n) ? n : 10;